src/components/Books.js

```diff
@@ -1,35 +1,28 @@
+import { useSelector, useDispatch } from 'react-redux';
 import Form from './form';
 import Book from './Book';
+import { addBook, removeBook } from '../redux/books/booksSlice';
 
 const Books = () => {
-  const books = [
-    {
-      id: 1,
-      title: 'Harry Potter and the Chamber of Secrets',
-      author: 'J.K. Rowling',
-    },
-    {
-      id: 2,
-      title: 'The Originals',
-      author: 'Caroline Kaplin',
-    },
-    {
-      id: 3,
-      title: 'The Firm',
-      author: 'Harvey Specter',
-    },
-    {
-      id: 4,
-      title: 'Game of Thrones',
-      author: 'Damon Salvator',
-    },
-  ];
+  const books = useSelector((state) => state.books.books);
+  const dispatch = useDispatch();
 
+  const handleRemoveBook = (bookId) => {
+      dispatch(removeBook({id: bookId}));
+      console.log(bookId);
+  }  
+
+  const handleAddBook = (item_id, author, title, e) => {
+    e.preventDefault();
+    dispatch(addBook({ item_id, author, title }));
+    console.log(item_id)
```

Comment:
- [ ] Kindly check y comment on line `12` concerning not using `console.log` in your code 👍